api/src/main/java/org/openmrs/module/locationbasedaccess/LocationBasedAccessActivator.java

```diff
@@ -53,9 +53,23 @@ public void started() {
 			    LocationBasedAccessConstants.PERSONATTRIBUTETYPE_UUID);
 	    createGlobalPropertyForLocationAttribute(LocationBasedAccessConstants.REF_APP_LOCATION_USER_PROPERTY_NAME,
 			    LocationBasedAccessConstants.LOCATION_USER_PROPERTY_NAME);
+	    createGlobalPropertyForEntityRestrictions();
 	    log.info("Location Based Access Control Module started");
     }
 
+	private void createGlobalPropertyForEntityRestrictions() {
```

Comment:
Can you add comments for this GP, then a developer can simply identify these use cases.

Reply:
Added comments.